meson.build

```diff
@@ -1717,6 +1717,15 @@ if conf.get('BPF_FRAMEWORK') == 1
                 '-c',
         ]
 
+        # If c_args contains prefix maps copy them along with the values, in order to avoid breaking
+        # reproducible builds
+        foreach opt : c_args
+                if opt.contains('-ffile-prefix-map=') or opt.contains('-fdebug-prefix-map=') or opt.contains('-fmacro-prefix-map=')
```

Comment:
.startswith?

Reply:
Not sure, could be quoted in weird ways perhaps?